src/tests/ftest/user_setup.py

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/env python3
+"""
+  (C) Copyright 2018-2022 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+
+import argparse
+import logging
+import os
+import sys
+
+from ClusterShell.NodeSet import NodeSet
+
+from util.logger_utils import get_console_handler
```

Comment:
I'm somewhat familiar with this error but don't know the solution.

All other python files in src/test/ftest/*.py which import from util hit the same problem, and I've had to suppress it.  The check passes for me locally but fails in github actions. I think part of the reason is there is a python module called "util" but from memory it's present both in my setup and in GHA so that doesn't entirely explain it.

Anyway, you should just suppress this error for now as not a regression and I'll take another look next time I'm doing a parse of the pylint code.


https://github.com/daos-stack/daos/blob/master/src/tests/ftest/config_file_gen.py#L8-L9
https://github.com/daos-stack/daos/blob/master/src/tests/ftest/launch.py#L30
https://github.com/daos-stack/daos/blob/master/src/tests/ftest/process_core_files.py#L14
https://github.com/daos-stack/daos/blob/master/src/tests/ftest/slurm_setup.py#L8